### DISCUSSION TITLE
Copy Project / Table for External DB connections [Jun 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5985] [Discord Threads]
> 🙋‍♂️ **lvarming**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Immediate roadmap, no.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1110079823986036806)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
